scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-customers-service.json

```diff
@@ -0,0 +1,153 @@
+  "networkMode": "awsvpc",
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "cpu": "256",
+  "memory": "512",
+  "containerDefinitions": [
+    {
+      "name": "customers-service",
+      "image": "customers-service-image",
+      "portMappings": [
+        {
+          "containerPort": 8081,
+          "protocol": "tcp"
+        }
+      ],
+      "environment": [
+        {
+          "name": "OTEL_EXPORTER_OTLP_PROTOCOL",
+          "value": "http/protobuf"
+        },
+        {
+          "name": "OTEL_LOGS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER",
+          "value": "xray"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+          "value": "http://localhost:4316/v1/traces"
+        },
+        {
+          "name": "OTEL_PROPAGATORS",
+          "value": "tracecontext,baggage,b3,xray"
+        },
+        {
+          "name": "OTEL_RESOURCE_ATTRIBUTES",
+          "value": "aws.log.group.names=/ecs/pet-clinic-customers-service,service.name=spring-petclinic-demo-customers-service"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT",
+          "value": "http://localhost:4316/v1/metrics"
+        },
+        {
+          "name": "OTEL_METRICS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "JAVA_TOOL_OPTIONS",
+          "value": " -javaagent:/otel-auto-instrumentation/javaagent.jar"
+        },
+        {
+          "name": "SPRING_PROFILES_ACTIVE",
+          "value": "ecs"
+        },
+        {
+          "name": "CONFIG_SERVER_URL",
+          "value": "http://config-server-cluster-name.ecs-petclinic:8888"
+        },
+        {
+          "name": "DISCOVERY_SERVER_URL",
+          "value": "http://discovery-server-cluster-name.ecs-petclinic:8761/eureka"
+        },
+        {
+          "name": "CUSTOMERS_SERVICE_IP",
+          "value": "customers-service-cluster-name.ecs-petclinic"
+        }
+      ],
+      "mountPoints": [
+        {
+          "sourceVolume": "opentelemetry-auto-instrumentation",
+          "containerPath": "/otel-auto-instrumentation",
+          "readOnly": false
+        }
+      ],
+      "essential": true,
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/pet-clinic-customers-service",
+          "awslogs-create-group": "true",
+          "awslogs-region": "region-name",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "cpu": 256,
+      "memory": 512
+    },
+    {
+      "name": "init",
+      "image": "adot-java-image",
+      "cpu": 0,
+      "portMappings": [],
+      "essential": false,
+      "command": [
+        "cp",
+        "/javaagent.jar",
+        "/otel-auto-instrumentation/javaagent.jar"
+      ],
+      "environment": [],
```

Comment:
nit: remove the fields as they are not mandatory.

Reply:
removed all no mandatory fields.